pkg/controller/conditions.go

```diff
@@ -129,14 +160,14 @@ func (d *VirtualMachineConditionManager) GetPodCondition(pod *k8sv1.Pod, conditi
 	return nil
 }
 
-func NewVirtualMachineInstanceConditionManager() *VirtualMachineConditionManager {
-	return &VirtualMachineConditionManager{}
+func VmiConditionManager() virtualMachineInstanceConditionManager {
```

Comment:
Is this change required for this PR?

Reply:
No, not required. I thought the names were too long. It is in a separate commit, should it be a separate PR too?

Reply:
I prefer it to go to a separate PR if possible since its unrelated.

Reply:
Ok, the other commit is removed.